sr_multi_moveit/sr_multi_moveit_test/test/test_planners.py

```diff
@@ -161,7 +168,7 @@ def test_waypoins(self):
         initial_pose = self.group.get_current_pose().pose
           
         initial_pose.position.x = -0.301185959729
-        initial_pose.position.y = 0.377069787724
+        initial_pose.position.y = 0.517069787724
```

Comment:
this is the only actual change. I added roughly the length of the forearm to make the waypoints reachable.